src/server/game/Globals/ObjectMgr.h

```diff
@@ -1403,6 +1415,16 @@ class TC_GAME_API ObjectMgr
             return nullptr;
         }
 
+        SceneTemplateContainer const& GetSceneTemplates() const { return _sceneTemplateStore; }
```

Comment:
Do not expose entire container if you don't need to (you are not calling this anywhere)